src/kaboom.ts

```diff
@@ -4022,30 +4094,34 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 		return game.ev.on("virtualButtonRelease", (b) => b === btn && action())
 	}
 
-	function onGamepadButtonDown(btn: string | ((btn: string) => void), action?: (btn: string) => void): EventController {
+	function onGamepadButtonDown(btn: GamepadButton | ((btn: GamepadButton) => void), action?: (btn: GamepadButton) => void): EventController {
 		if (typeof btn === "function") {
 			return game.ev.on("gamepadButtonDown", btn)
 		} else if (typeof btn === "string" && typeof action === "function") {
 			return game.ev.on("gamepadButtonDown", (b) => b === btn && action(btn))
 		}
 	}
 
-	function onGamepadButtonPress(btn: string | ((btn: string) => void), action?: (btn: string) => void): EventController {
+	function onGamepadButtonPress(btn: GamepadButton | ((btn: GamepadButton) => void), action?: (btn: GamepadButton) => void): EventController {
 		if (typeof btn === "function") {
 			return game.ev.on("gamepadButtonPress", btn)
 		} else if (typeof btn === "string" && typeof action === "function") {
 			return game.ev.on("gamepadButtonPress", (b) => b === btn && action(btn))
 		}
 	}
 
-	function onGamepadButtonRelease(btn: string | ((btn: string) => void), action?: (btn: string) => void): EventController {
+	function onGamepadButtonRelease(btn: GamepadButton | ((btn: GamepadButton) => void), action?: (btn: GamepadButton) => void): EventController {
 		if (typeof btn === "function") {
 			return game.ev.on("gamepadButtonRelease", btn)
 		} else if (typeof btn === "string" && typeof action === "function") {
 			return game.ev.on("gamepadButtonRelease", (b) => b === btn && action(btn))
 		}
 	}
 
+	function onGamepadStick(stick: "left" | "right", action: (value: Vec2) => void): EventController {
+		return game.ev.on("gamepadAxe", ((a: string, v: Vec2) => a === stick && action(v)))
```

Comment:
Nit: let's make the event name same as the function name `"gamepadStick"`

Reply:
For sure